pypeit/specutils/pypeit_loaders.py

```diff
@@ -35,6 +37,69 @@
 from pypeit import onespec
 
 
+def enforce_monotonic_wavelengths(wave, flux, ivar, strict=True):
+    """
+    Force the spectrum to have a monotonically increasing wavelength vector.
+
+    Parameters
+    ----------
+    wave : `numpy.ndarray`_
+        Spectrum wavelengths
+    flux : `numpy.ndarray`_
+        Spectrum flux
+    ivar : `numpy.ndarray`_
```

Comment:
should we make it an optional argument, then? `ivar=None`

Reply:
I think if this function were moved to somewhere in `core` to be used outside of this loader, then making this argument optional is fine.  Otherwise, this is just a helper function for use within this module and it doesn't matter since `ivar` in some form will be passed in all the time.

Reply:
Although @rcooke-ast 's suggestion is perfectly reasonable, I agree with @tbowers7 : this is really just a helper function that isn't intended to be used outside this module.  To signal this is a "private" function, I added a leading underscore to the function name.